### ISSUE TITLE
restructure the sherpa repository

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sherpa#17]
In IntelliJ IDEA, subdirectories of a repository can be excluded from a project, which is extremely useful for search/replace operations. Because all of the 3rd-party libraries are at the top-level of the sherpa repository, there is currently no way to exclude "unwanted" subdirectories while retaining "wanted" subdirectories.  This has resulted in recent changes being missed in info.js, breaking the build process.

How about if we put all of the 3rd-party libraries under sherpa/libs/? The repo structure would look like this:

```
sherpa/
  .gitignore   
  info.js
  LICENSE 
  README.md
  licenses/
    almond.txt
    ...
    webgl-debug-2012.txt
  libs/
    syntaxhighlighter-3.0.38/
    almond-0.2.9.js
    ...
    webgl-debug-2012.js
```
@samreid @jonathanolson @jbphet please weigh in.

**Comment by jonathanolson** [comment]
I like what you've proposed.

**Comment by jbphet** [comment]
+1

**Comment by samreid** [comment]
The proposed change seems reasonable, but I would like to note that I have never excluded sherpa from my searches and do not plan to exclude sherpa/libs from my searches.  Can you discuss why you have found it useful to exclude sherpa/?

Also, I have seen "lib" more than "libs" so would recommend it, but don't feel very strongly about that name.

**Comment by pixelzoom** [comment]
I exclude sherpa from searches so that I am not finding/replacing things in 3rd-party libraries.

**Comment by pixelzoom** [comment]
sherpa/lib/ is OK with me.

**Comment by pixelzoom** [comment]
> Can you discuss why you have found it useful to exclude sherpa/?

Similar to why you typically want to exclude build/ and node_modules/. I don't want to be finding/replacing things in 3rd-party libraries.

**Comment by pixelzoom** [comment]
Work on this is complete. All 3rd-party libs have been moved to sherpa/lib/.  I tested all development-mode HMTL files to verify that sims start (except for those that were broken before this refactor).  And I verified that all sims build by running `grunt-all.sh` (except for those that didn't build before this refactor). 

I did not verify unit tests (for sims and common code) because I'm not familiar with how to do that. (As an aside, it would be nice to have a script that runs all units tests.)  Assigning to @jonathanolson to verify unit tests, since he has written most (all?) of them.
